memory/memhotplug.py

```diff
@@ -38,6 +38,15 @@
             'double fault:', 'BUG: Bad page state in']
 
 
+def get_mem_count():
+    with open('/sys/devices/system/memory/block_size_bytes', 'r') as block_file:
+        blocksize = int(block_file.read()) / 10
```

Comment:
can you please help me to understand by only divide by 10 as here converting hex value to int than divide by 10 , can we right sift the value which is appropriate (divide by 2)   than assign to blocksize